File: Java/Day19.java
class Calculator implements AdvancedArithmetic{
    public int divisorSum(int n){
        int x=0;
        for(int i=1;i<=n;i++){
            if(n%i==0){
                x+=i;
            }
        }
        return x;
    }
    
}

File: Python/Day19.py

class Calculator implements AdvancedArithmetic{
    
    public function divisorSum($n){
       
        $sum=0;
        $i=0 ;
        for ($i=1;$i<$n;$i++){
            
            if($n%$i==0){
                $sum+=$i;
            }
        }
        
        return $sum+$n;
    }
    
    
    
    
}
